src/WebJobs.Script.WebHost/Security/HostSecrets.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public class HostSecrets
+    {
+        [JsonProperty(PropertyName = "masterKey")]
+        public Key MasterKey { get; set; }
+
```

Comment:
Will you ever need the ability to have two MasterKeys? What about if you have to roll it?